e2e-tests/portal/cypress/integration/journeys/maker/dashboard/facilities/dashboard-table-data.spec.js

```diff
@@ -0,0 +1,234 @@
+
+const {
+  BANK1_MAKER1,
+  BANK2_MAKER2,
+  ADMIN,
+} = MOCK_USERS;
+
+const {
+  BSS_DEAL,
+  GEF_DEAL,
+  BSS_DEAL_BANK_2_MAKER_2,
+} = MOCK_DEALS;
+
+const {
+  CASH_FACILITY,
+  BOND_FACILITY,
+} = MOCK_FACILITIES;
+
+const formatCurrencyValue = (value) =>
+  parseFloat(value).toFixed(2).replace(/\d(?=(\d{3})+\.)/g, '$&,');
+
+const hasBeenIssuedText = (hasBeenIssued) => {
+  if (hasBeenIssued) {
+    return 'Issued';
+  }
+
+  return 'Unissued';
+};
+
+context('View dashboard facilities as a maker', () => {
+  const ALL_DEALS = [];
+  let ALL_FACILITIES = [];
+  let ALL_BANK1_DEALS;
+  let ALL_BANK2_DEALS;
+  let gefDeal;
+  let gefFacility;
+  let gefFacilityId;
+  let bssDeal;
+  let bssFacility;
+  let bssFacilityId;
+
+  before(() => {
+    cy.deleteGefApplications(ADMIN);
+    cy.deleteDeals(ADMIN);
+
+    cy.listAllUsers().then((usersInDb) => {
+      const maker = usersInDb.find((user) => user.username === BANK1_MAKER1.username);
+      BSS_DEAL.maker = maker;
+    });
+
+    /*
+     * insert GEF deal and facility by bank 1, maker 1
+     */
+    cy.insertOneGefApplication(GEF_DEAL, BANK1_MAKER1)
+      .then((gefDeal) => {
+
+        cy.updateGefApplication(gefDeal._id, GEF_DEAL, BANK1_MAKER1)
+          .then((updatedGefDeal) => {
+            ALL_DEALS.push(gefDeal);
+          });
+
+        CASH_FACILITY.dealId = gefDeal._id;
+
+        cy.insertOneGefFacility(CASH_FACILITY, BANK1_MAKER1)
+          .then((gefFacility) => {
+            const { _id } = gefFacility.details;
+
+            cy.updateGefFacility(_id, CASH_FACILITY, BANK1_MAKER1)
+              .then((updatedGefFacility) => {
+                ALL_FACILITIES.push(updatedGefFacility.details);
+              });
+          });
+      });
+
+    /*
+     * insert BSS deal and facility by bank 1, maker 1
+     */
+    cy.insertOneDeal(BSS_DEAL, BANK1_MAKER1).then((bssDeal) => {
+      ALL_DEALS.push(bssDeal);
+
+      cy.createFacilities(bssDeal._id, [BOND_FACILITY], BANK1_MAKER1).then((createdFacilities) => {
+        ALL_FACILITIES = [
+          ...ALL_FACILITIES,
+          ...createdFacilities,
+        ];
+      });
+    });
+
+    /*
+     * insert BSS deal and facility by bank 2, maker 2
+     */
+    cy.insertOneDeal(BSS_DEAL_BANK_2_MAKER_2, BANK2_MAKER2).then((bssDeal) => {
+      ALL_DEALS.push(bssDeal);
+
+      cy.createFacilities(bssDeal._id, [BOND_FACILITY], BANK2_MAKER2).then((createdFacilities) => {
+        ALL_FACILITIES = [
+          ...ALL_FACILITIES,
+          ...createdFacilities,
+        ];
+      });
+    });
+  });
+
+  beforeEach(() => {
+    gefFacility = ALL_FACILITIES.find((facility) => facility.name.includes('GEF'));
+    gefFacilityId = gefFacility._id;
+    gefDeal = ALL_DEALS.find((deal) => deal.dealType === 'GEF');
+
+    bssFacility = ALL_FACILITIES.find((facility) => facility.name.includes('BSS'));
+    bssFacilityId = bssFacility._id;
+    bssDeal = ALL_DEALS.find((deal) => deal.dealType === 'BSS/EWCS');
```

Comment:
can we use constants here? - this is applicable for the lines above

Reply:
great stuff i've already done this